services/common/src/main/java/org/collectionspace/services/nuxeo/client/java/NuxeoFrameworkBootstrap.java

```diff
@@ -47,7 +48,11 @@ public void start(MutableClassLoader cl) throws ReflectiveOperationException, IO
             throw new IllegalStateException("Framework Loader was not initialized. Call initialize() method first");
         }
         Method start = frameworkLoaderClass.getMethod("start");
-        start.invoke(null);
+        try {
+            start.invoke(null);
+        } catch (BlobNotFoundException e) {
```

Comment:
Huh? This does seem to make sense.  Don't include this change.  BlobNotFoundException will never happen.